doc/rados/operations/add-or-rm-osds.rst

```diff
@@ -174,13 +174,13 @@ your new OSD before it can begin receiving data. You may use
 ``service ceph`` from your admin host or start the OSD from its host
 machine.
 
-For Debian/Ubuntu use Upstart. ::
+For Ubuntu Trusty use Upstart. ::
 
 	sudo start ceph-osd id={osd-num}
 
-For CentOS/RHEL, use sysvinit. ::
```

Comment:
Yeah that is what I was saying they would have to navigate through each branch, but generally the latest docs should handle atleast the last 2 releases? A new user or google redirected search would be misleading if he not aware of the current releases.